geoextent/lib/handleVector.py

```diff
@@ -47,7 +52,47 @@ def getTemporalExtent(filepath):
     input "path": type string, file path to vector file
     '''
 
-    return None
+    global parsed_time
+    dataset = ogr.Open(filepath)
+    layer = dataset.GetLayer()
+    layerDefinition = layer.GetLayerDefn()
+
+    field_names = []
+    for i in range(layerDefinition.GetFieldCount()):
+        field_names.append(layerDefinition.GetFieldDefn(i).GetName())
+
+    match_list = []
+    for x in search["time"]:
+        for j in field_names:
+            term = re.compile(x, re.IGNORECASE)
```

Comment:
Couldn't this compiling of the regular expressions be done _once_ and then a `field_names_regexes` be used?